### ISSUE TITLE
Cannot print

### ISSUE THREAD

**Issue by fauzanmh** [januslo/react-native-bluetooth-escpos-printer#28]
i have try printText("hello world") but printer can't print that text, you have solution?

**Comment by januslo** [comment]
plz check demo.

**Comment by mg4u** [comment]
I face the same problem too

**Comment by FazilMuhammed** [comment]
@fauzanmh any progress to printText("hello world").
**Me too same issue I have. my printer cannot printText **

Device iPhone.
I'm using Mini thermal printer.

**Comment by januslo** [comment]
Have you try my samples?

发自我的 iPhone

**Comment by FazilMuhammed** [comment]
yeah I tried your sample. I cannot print data in iOS, android working
properly.

initially check bluetooth is enable or disable. after search the
surrounding enabled devices and connect my printer device. after the
connect successful click the ESC/POS button to print.result is nothing
happening in iOS print.

Button Component

<Button disabled={this.state.loading || !(this.state.bleOpend && this.state.
boundAddress.length > 0 )}
title="ESC/POS"
onPress={ async () => {
console.log("Results")
try {
await BluetoothEscposPrinter.printerInit();
await BluetoothEscposPrinter.printerLeftSpace(0);
await BluetoothEscposPrinter.printerAlign(BluetoothEscposPrinter.ALIGN.
CENTER);
await BluetoothEscposPrinter.setBlob(0);
await BluetoothEscposPrinter.printText("Hello World\r\n", {
encoding: 'GBK',
codepage: 0,
widthtimes: 3,
heigthtimes: 3,
fonttype: 1
});
await BluetoothEscposPrinter.setBlob(0);
await BluetoothEscposPrinter.printText("Welcome\r\n", {
encoding: 'GBK',
codepage: 0,
widthtimes: 0,
heigthtimes: 0,
fonttype: 1
});
} catch (e) {
Alert.alert("OK1")
alert(e.message || "ERROR");
}
}}
/>

**Comment by aniltonveiga** [comment]
You need to change all encoding 'GBK' to 'UTF8' ;)
